### ISSUE TITLE
Allow generators to be supplied via GitHub URL, npm package or file

### ISSUE THREAD

**Issue by ColinEberhardt** [ScottLogic/openapi-forge#3]
The current implementation expects a generator to be provided via a filepath. It should support a range of options ...

via filepath:

```
openapi-forge forge ./petstore.json ../openapi-forge-typescript -o api
```

via npm package:

```
openapi-forge forge ./petstore.json openapi-forge-typescript -o api
```

via GitHub / git URL

```
openapi-forge forge ./petstore.json https//foo/openapi-forge-typescript -o api

**Comment by jhowlett-scottlogic** [comment]
@ColinEberhardt i would like to take on this issue too while I am working in the same area on #4.

**Comment by ColinEberhardt** [comment]
> i would like to take on this issue 

awesome - thanks.

**Comment by jhowlett-scottlogic** [comment]
@ColinEberhardt this issue has been resolved with PR#43
